### ISSUE TITLE
ae.fit()

### ISSUE THREAD

**Issue by ElijahMak** [daducci/AMICO#82]
Hello AMICO team,

First of all, thanks very much for putting this up! I ran into an error while running ae.fit and I wonder if you could help.

```
ae.fit()
-> Fitting "NODDI" model to 313630 voxels:
   [                                                  ]   0.0% Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "amico/core.py", line 324, in fit
    if self.niiMASK_img[ix,iy,iz]==0 :
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```
What could be wrong? Thanks!

**Comment by captainnova** [comment]
Hi, I'm not in the AMICO team, but I saw your question while checking to see if another issue had been updated. The error message you're getting is a very common one from numpy, and it means the array has a higher dimensionality than the expected scalar. It looks like self.niiMASK_img is reasonably enough expected to be 3D, but your mask is 4D or higher. Try checking that you did not make one mask for each volume in your data - making a mask from one of the volumes, or an average of the volumes, is what's expected.
